### DISCUSSION TITLE
Can you use Harpy without logging in?

### DISCUSSION THREAD

**Discussion by Ansem93** [robertodoering/harpy#714] [Q&A] [answered]
As the title sais, it would be possible to add an option to use harpy withou logging in? It would be helpful for people like me that just want to look around

**Comment by robertodoering** [comment, marked as answer]
That won't be possible since harpy is tightly tied to the twitter api which requires you to be authenticated.
I'd recommend https://github.com/zedeus/nitter as an alternative twitter frontend that doesn't require you to have a twitter account.
